src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
+
+    protected String dueDate;
+
+    public Deadline(String description, String dueDate) {
+        super(description);
+        this.dueDate = dueDate;
+    }
+
+    @Override
+    public String toString(){
```

Comment:
I think it is better to add a space before "{", although it is a very minor change.